ibm/service/cos/resource_ibm_cos_bucket_lifecycle_configuration_test.go

```diff
@@ -0,0 +1,1289 @@
+	serviceName := fmt.Sprintf("terraform_%d", acctest.RandIntRange(10, 100))
+	bucketName := fmt.Sprintf("terraform-lifecycle-configuration%d", acctest.RandIntRange(10, 100))
+	bucketRegion := "us-south"
+	bucketClass := "standard"
+	bucketRegionType := "region_location"
+	transitionDays := 1
+	tStorageclass := "Invalid"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMCosBucketDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccCheckIBMCosBucket_Lifecycle_Configuration_Transition_With_Days(serviceName, bucketName, bucketRegionType, bucketRegion, bucketClass, transitionDays, tStorageclass),
+				ExpectError: regexp.MustCompile(`"lifecycle_rule.0.transition.0.storage_class" must contain a value from \[\]string\{"GLACIER", "ACCELERATED"\}, got "Invalid`),
+			},
+		},
+	})
+}
+func TestAccIBMCosBucket_Lifecycle_Configuration_Transition_Multiple_Rules(t *testing.T) {
+	serviceName := fmt.Sprintf("terraform_%d", acctest.RandIntRange(10, 100))
+	bucketName := fmt.Sprintf("terraform-lifecycle-configuration%d", acctest.RandIntRange(10, 100))
+	bucketRegion := "us-south"
+	bucketClass := "standard"
+	bucketRegionType := "region_location"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMCosBucketDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccCheckIBMCosBucket_Lifecycle_Configuration_Transition_With_Multiple_Rules(serviceName, bucketName, bucketRegionType, bucketRegion, bucketClass),
+				ExpectError: regexp.MustCompile("MalformedXML: The XML you provided was not well-formed or did not validate against our published schema."),
+			},
+		},
+	})
+}
+
+// check
+func testAccCheckIBMCosBucket_Lifecycle_Configuration_Expiration_With_Days(cosServiceName string, bucketName string, regiontype string, region string, storageClass string, days int) string {
+
+	return fmt.Sprintf(`
+	data "ibm_resource_group" "cos_group" {
+		name = "Default"
+	}
+
+	resource "ibm_resource_instance" "instance" {
+		name              = "%s"
+		service           = "cloud-object-storage"
+		plan              = "standard"
+		location          = "global"
+		resource_group_id = data.ibm_resource_group.cos_group.id
+	}
+	resource "ibm_cos_bucket" "bucket" {
+		bucket_name           = "%s"
+		resource_instance_id  = ibm_resource_instance.instance.id
+	    region_location = "%s"
+		storage_class         = "%s"
+	}
+	resource "ibm_cos_bucket_lifecycle_configuration"  "lifecycle" {
+		bucket_crn = ibm_cos_bucket.bucket.crn
+		bucket_location = ibm_cos_bucket.bucket.region_location
+		lifecycle_rule {
+		   expiration{
+		     days = "%d"
+		   }
+		   filter {
+		     prefix = "foo"
+		   }  
+		   rule_id = "id"
+		   status = "Enabled"
+	
+		 }
+	  }
+	
+	 
```

Comment:
extra blank line. Fix it for all instances in this file

Reply:
done